### PR TITLE
Add drop hints script from Noto

### PR DESCRIPTION
**Pull request by simoncozens**

### CODE REVIEW

File: bin/gftools-drop-hints.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""Drop hints from a font."""
+
+__author__ = "roozbeh@google.com (Roozbeh Pournader)"
```
Comment (m4rc1e) [inline bin/gftools-drop-hints.py:20]: He left a while ago so I don't think that email works anymore.

File: bin/gftools-drop-hints.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    """Drops the listed tables from a font."""
+    for table in tables:
+        if table in font:
+            del font[table]
+
+
+def main():
+    """Drop the hints from the first file specified and save as second."""
+    args = parser.parse_args()
+    font = ttLib.TTFont(args.input)
```
Comment (m4rc1e) [inline bin/gftools-drop-hints.py:52]: `ttLib.TTfont` --> `TTFont`

**Comment by m4rc1e** [comment]
Lemme change these since they're small.

File: bin/gftools-drop-hints.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+parser.add_argument("output", help="file to save font", nargs="?")
+
+
+def drop_hints_from_glyphs(font):
+    """Drops the hints from a font's glyphs."""
+    glyf_table = font["glyf"]
+    for glyph_index in range(len(glyf_table.glyphOrder)):
+        glyph_name = glyf_table.glyphOrder[glyph_index]
+        glyph = glyf_table[glyph_name]
+        if glyph.numberOfContours > 0:
```
Comment (anthrotype) [inline bin/gftools-drop-hints.py:34]: This is incomplete for it only drops instructions for simple glyphs, but not also for components in composite glyphs (where numberOfContours is -1)

Have a look at fonttools subsetter

**Pull request merged**